tasks/main.yml

```diff
@@ -2,23 +2,32 @@
 - name: Setup firewalld
   include_tasks: firewalld.yml
 
-- name: Collect service facts
-  service_facts:
```

Comment:
Looks like we cannot use service_facts - a quick grep shows that we use it in 25 different places in our roles.

Could we use `systemctl list-unit-files --no-pager --type service --all` to somehow scan for unit files, then is-enabled on those?

We might need our own system roles `lsr_service_facts` module that we could use with all of our roles.

Reply:
I considered `systemctl list-unit-files --no-legend --type service`, initially, but I found the rather elaborate string parsing and making assumptions about the output format more brittle than using a direct `is-enabled`  test -- that will only happen at most three times, so it's not very expensive. I can rewrite it if you feel strongly about it, of course.

Reply:
Yes, agreed -- if it's more than just a handful, then a summary `list-unit-files` is better. I hope not many roles need that, though.